lib/flapjack/gateways/email.rb

```diff
@@ -60,19 +60,36 @@ def start
         end
       end
 
+      def safe_address(addr)
+        return "flapjack@#{@fqdn}" if addr.nil? || addr.empty?
+        safe_addr = addr.clone
+        while safe_addr =~ /(<|>)/
+          safe_addr.sub!(/^.*</, '').sub!(/>.*$/, '')
+        end
+
+        safe_addr
+      end
+
       def deliver(alert)
-        host = @smtp_config ? @smtp_config['host'] : nil
-        port = @smtp_config ? @smtp_config['port'] : nil
-        starttls = @smtp_config ? !! @smtp_config['starttls'] : nil
-        m_from = @smtp_config ? @smtp_config['from'] : "flapjack@#{@fqdn}"
-        m_reply_to = @smtp_config ? ( @smtp_config['reply_to'] ||= m_from ) : "flapjack@#{@fqdn}"
         if @smtp_config
+          host = @smtp_config['host']
+          port = @smtp_config['port']
+          starttls = !!@smtp_config['starttls']
+
+          m_from = safe_address(@smtp_config['from'])
+          m_reply_to = @smtp_config['reply_to'] || @smtp_config['from']
+
           if auth_config = @smtp_config['auth']
             auth = {}
-            auth[:type]     = auth_config['type'].to_sym || :plain
+            auth[:type] = auth_config['type'].to_sym || :plain
```

Comment:
@jessereynolds Done. Note that is the alignment from master, so I assumed that was the preferred style (nowhere else either around that code seems like assignment operators are aligned like that).